features/bevragen/persoon/adressering/aanhef/summary.feature

```diff
@@ -1,16 +1,129 @@
+# language: nl
+
 Functionaliteit: Als gemeente wil ik de juiste en consistente briefaanhef in communicatie naar burgers
```

Comment:
Dit is naar mijn mening niet de benaming voor een functionaliteit. Ik zou eerder denken aan: aanhef bepaling. Brief is weggelaten omdat de aanhef ook kan worden gebruikt in e-mail

Reply:
Aangepast